### PR TITLE
Added a paragraph to the story and some ASCII art

### PR DESCRIPTION
**Pull request by floridop**
Added some content to your story. You are free to ask me to do any change you like before approving!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
 # MNXB01-learn
 
-Once upon a time, in a land closer than you think, there lived a **rich** but _cruel_ king - The Coldran of Kor. He was feared by all his subject and everyone who had the unfortunate chance to hear his name spoken.
+Once upon a time, in a land closer than you think, 
+there lived a **rich** but _cruel_ king - The Coldran of Kor.
```
Comment (udi251nikolei) [inline README.md]: I'd like to follow up line 3 with line 4 after the comma.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,32 @@
 # MNXB01-learn
 
-Once upon a time, in a land closer than you think, there lived a **rich** but _cruel_ king - The Coldran of Kor. He was feared by all his subject and everyone who had the unfortunate chance to hear his name spoken.
+Once upon a time, in a land closer than you think, 
+there lived a **rich** but _cruel_ king - The Coldran of Kor. 
+He was feared by all his subject and everyone who had the unfortunate 
+chance to hear his name spoken.
```
Comment (udi251nikolei) [inline README.md]: I don't want line 6 to be a separated from line 5. Follow up line 5 with line 6.

**Review by udi251nikolei — CHANGES_REQUESTED** [review]
Thanks for your contribution. I saw you have made some edits on my first draft. I've written instructions in the 2 comments I've provided on specific line I want changes.

**Comment by floridop** [comment]
Requested changes implemented in commit [0c7968b](https://github.com/udi251nikolei/MNXB01-learn/pull/2/commits/0c7968bc7c518222d3a9c9a6bbe699ff2ed4bf25)

**Comment by floridop** [comment]
Hello! could you please check that you're fine with the changes and eventually merge?

**Comment by floridop** [comment]
Information about merging when you're the release manager is in Tutorial 5 slides 78-82

**Comment by udi251nikolei** [comment]
Merging completed

**Comment by floridop** [comment]
I'll close this part of the homework, but if there is something else 
you're curious to try and I am available I'll be happy to test.

**Pull request merged**